ambassador/03-ambassador-service.yaml

```diff
@@ -40,8 +40,10 @@ spec:
   ports:
    - name: http
      port: 80
+     nodePort: 31080
      targetPort: 8080
    - name: https
+     nodePort: 31443
```

Comment:
Will these changes disrupt non-Kubernaut users, e.g., folks using a standard LB?

Reply:
@plombardi89 correct me if I am wrong but I think that would only affect users if they happen to have another `NodePort` service using those ports.

Reply:
It may be a problem for folks using an existing cluster. It could impact someone with a `LoadBalancer` service since that is just an abstraction built on top of a `NodePort` and if the configured `NodePort` is in use then there will be a conflict.

I did it for convenience and simplicity. If we want to change it back to random the docs need to be modified appropriate to tell folks to get the port AND the IP.

Reply:
I'm going to revert it back in the off chance someone runs into the issue